packages/cli/test/commands/pull.test.ts

```diff
@@ -31,8 +72,8 @@ describe('pull', () => {
       id: 'vercel-pull-next',
       name: 'vercel-pull-next',
     });
-    const expectedEnvFilename = '.env.vercel';
-    client.setArgv('pull', '--yes', `--env=${expectedEnvFilename}`, cwd);
+    const expectedEnvFilename = '.env.vercel.development.local';
+    client.setArgv('pull', '--yes', '--env=.env.vercel', cwd);
```

Comment:
Does the `--env` option make sense to include anymore? It seems like it was an option put in place to allow users to move the file, but we're about to stuff all of these into the `.vercel` directory.